localstack/services/edge.py

```diff
@@ -427,6 +430,35 @@ def start_dns_server(asynchronous=False):
         pass
 
 
+def start_proxy(port, asynchronous=False):
```

Comment:
nit: wouldn't it be better to make the host and destination parameters of the function and let the caller set the config? not a big deal though.

Reply:
I think there might be a bit of a confusion here. This function (`start_proxy`) is the function which starts the new "proxy" edge component (similar to "dns" or "edge"). So the purpose of this function here is exactly to prepare the parameters for the underlying `start_tcp_proxy` and call it.

Reply:
gotcha